app/Http/Controllers/Auth/LoginController.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace App\Http\Controllers\Auth;
+namespace App\Http\s\Auth;
```

Comment:
This change causes the error when a User tries to log in. I guess it was just a typo because I don't see its relevance. 
```suggestion
namespace App\Http\Controllers\Auth;
```

Reply:
@reifred Thank you for pointing that out and I have fixed this as well.